tests/rust/pipe-test/open_pipe/mod.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//======================================================================================================================
+// Imports
+//======================================================================================================================
+
+use ::anyhow::Result;
+use ::demikernel::{
+    LibOS,
+    QDesc,
+};
+
+//======================================================================================================================
+// Standalone Functions
+//======================================================================================================================
+
+/// Drives integration tests for pipe queues.
+pub fn run(libos: &mut LibOS, pipe_name: &str) -> usize {
+    let mut nfailed: usize = 0;
+
+    nfailed += crate::test!(open_pipe_with_invalid_name(libos));
+
+    nfailed
+}
+
+/// Attempts to open a pipe with an invalid name.
+fn open_pipe_with_invalid_name(libos: &mut LibOS) -> Result<()> {
+    // Fail to create pipe with an invalid name.
+    match libos.create_pipe(&format!("")) {
+        Err(e) if e.errno == libc::EINVAL => Ok(()),
+        Ok(_) => anyhow::bail!("create_pipe() with invalid name should fail"),
+        Err(e) => anyhow::bail!("create_pipe() failed with {}", e.cause),
```

Comment:
I started leaving these as just e because I thought it might be nice to print the error code too and because our Fails can print both. You might consider doing that here, rather than throwing away the error code.

Reply:
Sure, I'll change this.